kitsune/users/migrations/0005_set_initial_contrib_email_flag.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""
+Set the `first_{revision,question}_email_sent` flags for existing users, to
```

Comment:
Seems like this doesn't affect "existing users", but rather more subtly "existing users that have contributed revisions and answers already". Does that sound right? If so, I think the docstring should be clarified accordingly.